backend/plugin/parser/cassandra/query_span_extractor.go

```diff
@@ -60,6 +64,13 @@ func (e *querySpanExtractor) EnterSelect_(ctx *cql.Select_Context) {
 	results := e.extractSelectElements(ctx.SelectElements(), keyspace, table)
 	e.querySpan.Results = results
 
+	// Add all result columns to SourceColumns (simplified implementation)
+	for _, result := range results {
+		for col := range result.SourceColumns {
+			e.querySpan.SourceColumns[col] = true
+		}
+	}
+
```

Comment:
Generally speaking, the sourceColumns in query span are used by Bytebase to check access tables. At the same time, we want it to be effective even when there is no schema metadata, so we usually use an independent function to handle this functionality here, rather than making it a result that depends on the query span result. Regarding this point, you can refer to how other engines implement it.